src/spawn.c

```diff
@@ -694,7 +694,8 @@ static gboolean spawn_async_with_pipes(const gchar *working_directory, const gch
 			for (argc = 0; argv[argc]; argc++);
 
 		full_argv = g_renew(gchar *, cl_argv, cl_argc + argc + 1);
-		memcpy(full_argv + cl_argc, argv, argc * sizeof(gchar *));
+		if (argv)
```

Comment:
@elextr Thoughts?

Reply:
I only language lawyer C++.

Perhaps it would be better to reference memcpy() docs:
```
If either dest or src is an invalid or null pointer, the behavior is undefined, even if count is zero.
```
so this is ok unless somebody finds a problem with it.